### PR TITLE
Update graphs.md

### PR DESCRIPTION
**Pull request by sbromberger**
Update for GSOC '20

### CODE REVIEW

**Comment by sbromberger** [comment]
cc @matbesancon @simonschoelly @jpfairbanks @somil55

File: content/jsoc/gsoc/graphs.md [hunk 1]
```diff
@@ -5,51 +5,36 @@ title:  JuliaGraphs – Summer of Code
 
 # {{< get_param title >}}
 
+Thank you for your interest in supporting JuliaGraphs in the upcoming Summer of Code!
```
Comment (matbesancon) [inline content/jsoc/gsoc/graphs.md]: Not sure this part is needed, it's a presentation page and people might not (yet) be interested in supporting lg

Comment (sbromberger) [reply content/jsoc/gsoc/graphs.md]: They've navigated to this page, so they're (possibly) interested in supporting us. I don't think it's a bad idea to reinforce this feeling :)

**Comment by sbromberger** [comment]
Thank you!

**Pull request merged**